### ISSUE TITLE
VCF quickcheck

### ISSUE THREAD

**Issue by yashpatel6** [uclahs-cds/public-tool-PipeVal#76]
Add option to do a quick check for VCF files. Potentially through PySAM?

**Comment by tyamaguchi-ucla** [comment]
Yup, I think we want to make it default and keep the current full check as an option.
